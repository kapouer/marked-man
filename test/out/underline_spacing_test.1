UNDERLINE_SPACING_TEST()                              UNDERLINE_SPACING_TEST()

NNAAMMEE
       uunnddeerrlliinnee__ssppaacciinngg__tteesstt

              • --SS _t_e_x_t, sseeaarrcchh _t_e_x_t:
                Performs a substring search of formula names for _t_e_x_t.

              • --OO <text>, ootthheerr <text>:
                Does something else.

       Should space text properly.  <!-- this is a comment -->

                                 January 1979         UNDERLINE_SPACING_TEST()
