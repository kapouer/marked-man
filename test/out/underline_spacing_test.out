[4mUNDERLINE_SPACING_TEST[24m(1)   General Commands Manual  [4mUNDERLINE_SPACING_TEST[24m(1)

[1mNAME[0m
       [1munderline_spacing_test[0m

        â€¢ [1m-S [4m[22mtext[24m, [1msearch [4m[22mtext[24m:
          Performs a substring search of formula names for [4mtext[24m.

        â€¢ [1m-O [22m<text>, [1mother [22m<text>:
          Does something else.

       Should space text properly.  <!-- this is a comment -->

                                 January 1979        [4mUNDERLINE_SPACING_TEST[24m(1)
